dependabot/resources/extensions.json

```diff
@@ -691,6 +691,19 @@
             "code_owner_id_env": "KALAIYARASI_USER_ID",
             "dependents": []
         },
+        {
```

Comment:
This is autogenerated file, no need to change here.

Reply:
Fixed in c923a0f